.github/workflows/release.yml

```diff
@@ -1,9 +1,16 @@
 name: release
 
 on:
-  push:
-    tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+*'
+  workflow_dispatch:
+    inputs:
+      versionNumber:
+        description: 'Release version number'
+        type: string
+        required: true
+
+env:
+  CI_COMMIT_AUTHOR: hc-github-team-tf-provider-devex
+  CI_COMMIT_EMAIL: github-team-tf-provider-devex@hashicorp.com
```

Comment:
We should consider moving these to [configuration variables](https://github.blog/changelog/2023-01-10-github-actions-support-for-configuration-variables-in-workflows/) in the future.

Reply:
Created an [issue](https://github.com/hashicorp/terraform-providers-devex-internal/issues/129) to track this.